catalyst_voices/lib/pages/account/account_page.dart

```diff
@@ -0,0 +1,225 @@
+import 'package:catalyst_voices/widgets/buttons/voices_icon_button.dart';
+import 'package:catalyst_voices/widgets/buttons/voices_text_button.dart';
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_brands/catalyst_voices_brands.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+
+final class AccountPage extends StatelessWidget {
+  const AccountPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SingleChildScrollView(
+        child: Padding(
+          padding: const EdgeInsets.all(32),
+          child: Column(
+            children: [
+              _Header(),
+              Padding(
+                padding: const EdgeInsets.symmetric(horizontal: 16),
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    const SizedBox(height: 32),
+                    _Tab(),
+                    const SizedBox(height: 48),
+                    _KeychainCard(
+                      connectedWallet: 'Lace',
+                      roles: const [
+                        'Voter (Default)',
+                        'Proposer',
+                        'Drep',
```

Comment:
These roles are known and pretty stable, I would extract them as l10n keys.

Reply:
Even enum would be cool. As we have for spaces